src/Microsoft.AspNetCore.Authentication.OAuth/Claims/ClaimActionCollectionMapExtensions.cs

```diff
@@ -96,5 +105,23 @@ public static void DeleteClaim(this ClaimActionCollection collection, string cla
         {
             collection.Add(new DeleteClaimAction(claimType));
         }
+
+        /// <summary>
+        /// Delete all claims from the ClaimsIdentity with the given claimTypes.
+        /// </summary>
+        /// <param name="collection"></param>
+        /// <param name="claimTypes"></param>
+        public static void DeleteClaims(this ClaimActionCollection collection, params string[] claimTypes)
```

Comment:
Delete?

Reply:
People have been very confused by Remove vs Delete. We need to avoid aggravating that.

Reply:
you could also just change DeleteClaim to take a params string[] right?

Reply:
In theory, but it would be binary breaking.